enterprise/config/buildbuddy.local.yaml

```diff
@@ -7,6 +7,8 @@ app:
   enable_target_tracking: true
 database:
   data_source: "sqlite3:///tmp/buildbuddy-enterprise.db"
+clickhouse:
```

Comment:
I don't think we should tie the naming to a specific implementation. What if we call this 'olap_database' as an analog to 'database'?